src/core/execute.c

```diff
@@ -730,14 +730,20 @@ static int restore_confirm_stdio(int *saved_stdin, int *saved_stdout) {
         return r;
 }
 
-static int ask_for_confirmation(const char *vc, const char *cmdline, char *response) {
+static int ask_for_confirmation(const char *vc, const Unit *u, const char *cmdline, char *response) {
         int saved_stdout = -1, saved_stdin = -1, r;
 
         r = setup_confirm_stdio(vc, &saved_stdin, &saved_stdout);
         if (r < 0)
                 return write_confirm_error(r, vc);
 
-        r = ask_char(response, "yns", "Execute %s? [Yes, No, Skip] ", cmdline);
+ask_again:
+        r = ask_char(response, "ynvs", "Start %s ? [Yes, No, View, Skip] ", u->id);
+        if (r >= 0 && *response == 'v') {
+                printf("  Description : %s\n", u->description);
+                printf("  ExecStart   : %s\n", cmdline);
```

Comment:
in most other outputs like this we place the colon right after the field name, and the additoinal whitespace after it..

Also, I think these two printf() invocations should become one.